test/pytest/test_example_intel_extension_for_pytorch.py

```diff
@@ -88,6 +88,10 @@ def scale_workers_with_core_pinning(scaled_num_workers):
     or ((torch.cuda.device_count() > 0) and torch.cuda.is_available()),
     reason="Make sure intel-extension-for-pytorch is installed and torch.backends.xeon.run_cpu is available",
 )
+@pytest.mark.skipif(
```

Comment:
Could you please create an issue tracker with the skipped tests, these tests are not great but they're all important dependencies ONNX, worker thread affinity and compile

Reply:
Sure, will create and update the PR with issues. But, we cannot test ONNX because of NVIDIA base image

Reply:
I intend to address this when I update Dockerfile.dev, where we can possibly build with nvidia runtime image

Reply:
Just noticed that ONNX test case is passing in docker :D (It shouldn't)